apps/sr-frontend/src/app/App.tsx

```diff
@@ -28,6 +33,14 @@ export default function App() {
               <Stack gap={1} pt={1}>
                 <CharacterSelector />
                 <Character />
+                <Accordion>
```

Comment:
Wrap this `<Accordion>` component inside a `<Container>` component to make it not full-width.

Reply:
fixed on 3a3a79cfa8aecc6d8a03fffe8ef070ab99b731a3